pep/app/pods/application/controller.ts

```diff
@@ -7,10 +7,27 @@ import SessionService from 'ember-simple-auth/services/session';
 
 import LoadingBarService from 'pep/services/loading-bar';
 import { SEARCH_TYPE_EVERYWHERE, SearchTermValue } from 'pep/constants/search';
+import AjaxService from 'pep/services/ajax';
+import NotificationService from 'ember-cli-notifications/services/notifications';
+
+import Modal from '@gavant/ember-modals/services/modal';
+import ENV from 'pep/config/environment';
+export interface ServerStatus {
```

Comment:
not necessary now, but maybe the best thing would be to have like a `types/api/..` file for these kinds of interfaces, especially if we end up having other raw ajax request endpoints we hit later that need their own interfaces too.